src/handlers/activities.js

```diff
@@ -96,14 +100,17 @@ class ActivitiesHandler extends BaseHandler {
     const rivers = (await riversApi.list(request))
       .filter(r => process.env.CONTEXT === 'FMT' ? true : !r.internal).sort(riversApi.sort)
 
+    // If it's a leap year, set the max number of days fished to 168 instead of 167
+    const maxDaysFished = submission.season % 4 === 0 ? MAX_DAYS_LEAP_YEAR : MAX_DAYS_NON_LEAP_YEAR
```

Comment:
Not sure if you realise this, but this isn't the whole of the formula for determining a leap year. It's a close enough approximation for our purposes, but the full rules are that a leap year is: 
* Divisible by 4: If a year is evenly divisible by 4, it's a leap year.
* Not divisible by 100: If a year is also evenly divisible by 100 (a century year), it is not a leap year.
* Divisible by 400: However, if a year is divisible by both 100 and 400 (and 4), it is a leap year.
No need to change this, as I doubt the current version of RCR will still be in use in 2100 which is divisible by 4, but isn't a leap year as it's also divisible by 100.